### ISSUE TITLE
Opera does not load the call

### ISSUE THREAD

**Issue by danieleds** [signalwire/browser-videoconf-full-react#8]
![image](https://user-images.githubusercontent.com/79721020/162197638-1aa401d4-b815-42e0-804b-26a05763bf1a.png)

I tested Chrome, Edge, Brave successfully but Opera hangs forever when trying to join the conference.

**Comment by sparkplug0025** [comment]
This is my Opera version details:

```
Opera GX
LVL3 (core: 84.0.4316.52)

Browser identification
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.109 Safari/537.36 OPR/84.0.4316.52
```

**Comment by danieleds** [comment]
Hi @sparkplug0025,

we could reproduce this on Opera. However, in that case Google Hangout didn't work either, asking us to grant camera permissions even though they were already granted:

<img width="622" alt="Screenshot 2022-04-13 at 15 05 29" src="https://user-images.githubusercontent.com/402652/163186726-0aa93417-fbce-44bc-90f3-40e48f38de41.png">

After I disconnected and reconnected one of my camera devices, and restarted Opera, then it started working fine.

A couple of things we can try:
 - Could you try again with SignalWire's demo and Google Hangout both open in parallel in two different tabs, to make sure that the behavior is consistent?
 - Could you share the output of the following instructions, executed from the JavaScript inspector? (any tab is fine)
   - `await navigator.mediaDevices.enumerateDevices()`
   - `await navigator.mediaDevices.getUserMedia({ audio: true, video: true })`

**Comment by danieleds** [comment]
It appears that the issue was caused by a problem in Opera. We have released v3.11.0 of the js sdk to trigger an error message after a timeout.
